ci/scripts/assume-role.js

```diff
@@ -8,7 +8,8 @@ const sts = new AWS.STS()
 const run = async function run () {
   const assumeRoleResponse = await sts.assumeRole({
     RoleArn: process.env.AWS_ROLE_ARN,
-    RoleSessionName: process.env.AWS_ROLE_SESSION_NAME
+    RoleSessionName: process.env.AWS_ROLE_SESSION_NAME,
+    DurationSeconds: 10800, // 3 hours
```

Comment:
We should take this as an env var and in the assume-role task take the duration as a param so we can override it just for our perf test.

I want us to actually merge these changes permanently since it's always going to be an issue for longer perf tests).